tests/cypress/integration/devportal/000-general/03-self-signup.spec.js

```diff
@@ -21,10 +21,15 @@ function getSuperTenantEmail(username) {
     return `${username}@test.com`;
 }
 
+function getTenantUser(username, domain) {
```

Comment:
Shall we move this code into a util class?

Reply:
Moved in https://github.com/wso2/apim-apps/pull/375/commits/01d02797e5a18f82c29825c51b653442ed99d964